src/xdocs/config_whitespace.xml

```diff
@@ -1630,6 +1632,15 @@ foo(i,
     s);
         </source>
         <p>
+          Example for method reference (good case and bad case):
+        </p>
+        <source>
+Arrays.sort(stringArray, String:: // violation, breaking after method reference is bad practice
```

Comment:
It is not a general bad practice. User is able to decide whether he/she wants to configure the check to rise the violation if ```::``` is on the next line after line break. Please remove ```breaking after method reference is bad practice```.
Please, add an example of check's configuration as METHOD_REF is not in the set of default tokens.